src/node_mgmt/env_scan.py

```diff
@@ -52,32 +52,40 @@
 
 SERIAL_SCAN_SIGNATURES = [{
     'name': 'Gamicos ultrasonic sensor',
-    'readings': [{
+    'slave_id': 1,
+    'readings': {
```

Comment:
I wouldn't call this `readings` given that it's not an array but just a single reading. In fact I'm not sure we even need this level in the hierarchy (given that the "signature" is now just a single reading). We could just have each signature be an object with the following items: `name`, `slave_id`, `reading`, `register`, `words`, `fncode`.